### PR TITLE
Add Past Events Functionality

### PR DESCRIPTION
**Pull request by Tcintra**
Create servlets and cron jobs to handle the transformation of approved and unapproved events once their date is passed. Please disregard the first 12 commits, I had problems rebasing my branch and I think some of the commits from previous pull requests got mixed up with this pull request.

### CODE REVIEW

File: project/src/main/java/com/google/sps/servlets/Announcements.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              ArrayList<Object> pair = new ArrayList<Object>();
+              pair.add((String) entity.getProperty("from"));
+              pair.add((String) entity.getProperty("title"));
+              pair.add((String) entity.getProperty("content"));
+              data.add(pair);
+          }
+      }
+      response.setContentType("application/json;");
+      response.getWriter().println(gson.toJson(data));
+      //this will get all announcements entities, with school = current user school
```
Comment (wulian1223) [inline project/src/main/java/com/google/sps/servlets/Announcements.java:95]: make the comment on the top?

Comment (wulian1223) [reply project/src/main/java/com/google/sps/servlets/Announcements.java:95]: and where did you parse and use "current school"?

Comment (Tcintra) [reply project/src/main/java/com/google/sps/servlets/Announcements.java:95]: For some reason my git fetch ended up pulling Fernando's previous branch. I did not implement the announcements.java file... I am not sure why it ended up in my pull request.

Comment (Tcintra) [reply project/src/main/java/com/google/sps/servlets/Announcements.java:95]: There are five commits that somehow got intertwined with Fernando's pull request

**Review by christinachance — APPROVED** [review]

File: project/src/main/java/com/google/sps/servlets/Announcements.java (hunk 1 above)
Comment (wulian1223) [reply project/src/main/java/com/google/sps/servlets/Announcements.java:95]: ok. Please make sure you only merge the last 3 commits.

**Review by wulian1223 — APPROVED** [review]
LGTM. Please make sure only merge your commits.

**Review by rivera-fernando — APPROVED** [review]

**Pull request closed without merging**